src/components/PopupRevisionOverview/PopupRevisionOverview.js

```diff
@@ -19,9 +19,11 @@ import useLockBodyScroll from './../../utils/useLockBodyScroll'
 import UserContext from './../../App/UserContext'
 
 /**
+ * Function to check the text within the object if it's valid.
```

Comment:
Function to check the text within the object if it's valid.
 >
Function te get the text indicating its validity

Reply:
I've changed the function description text.